File: 2/even_odd.c
int isEven(int num)
{
    // (num & 1) is 1, !(1)= 0 >> odd, else even
    return (!(num & 1));
}

int main()
{
    int num;
    printf("Enter the number: ");
    scanf("%d",&num);

    isEven(num)? printf ("Even"):printf ("Odd");
    return 0;
}

File: 2/even_odd.java
import java.util.Scanner;

public class even_odd{
    public static void main (String[]args){
        System.out.println("Enter the number: ");
        Scanner scan = new Scanner(System.in);
        int n= scan.nextInt();
        scan.close();

        String status = n % 2 == 0 ? " is Even" : " is Odd";
        System.out.println (n + status);
      }
    
    }

